### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by alhamilton1111**
Keeping the community harmonious and free of harassment, trolling, or spamming.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.22.0] - 2022-11-21
+
+### Added
+-   Updated the documentation to include a Code of Conduct
+
```
Comment (stephanebruckert) [inline CHANGELOG.md:12]: move to unreleased

File: setup.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 setup(
     name='spotipy',
-    version='2.21.0',
+    version='2.22.0',
```
Comment (stephanebruckert) [inline setup.py:21]: keep 2.21.0 please

Comment (alhamilton1111) [reply setup.py]: Ok will do, thanks for your assistance!

File: CHANGELOG.md (hunk 1 above)
Comment (alhamilton1111) [reply CHANGELOG.md:11]: Done, thanks!

**Pull request closed without merging**